analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -227,16 +229,30 @@ def get_checker_list(self, config) -> Tuple[List[str], List[str]]:
             state, _ = value
 
             # Checker name is a compiler warning.
+            if checker_name.startswith('W'):
```

Comment:
I feel like this branch should be moved to `get_compiler_warning_name`, that function is the authority on what we interpret to be a compiler warning.

Reply:
I thought about this, but `get_compiler_warning_name` is not a private member. In the future somebody could call it in some other context, then it would warning even if it is not related.